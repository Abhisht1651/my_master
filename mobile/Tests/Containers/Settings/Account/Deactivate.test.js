// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import Deactivate from '../../../../App/Containers/Settings/Account/Deactivate'

describe('Deactivate', () => {
  it('Deactivate component renders correctly', () => {
    const tree = shallow(<Deactivate />)
    expect(tree).toMatchSnapshot()
  })

  it('should render the Text Component', () => {
    const wrapper = shallow(<Deactivate />)
    expect(wrapper.find('Text')).toHaveLength(3)
  })

  it('should render the Image Component', () => {
    const wrapper = shallow(<Deactivate />)
    expect(wrapper.find('Image')).toHaveLength(1)
  })
})
