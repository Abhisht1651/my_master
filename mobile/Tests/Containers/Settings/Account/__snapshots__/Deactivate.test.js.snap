// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deactivate Deactivate component renders correctly 1`] = `
<View
  style={
    Object {
      "zIndex": 1,
    }
  }
>
  <View
    style={
      Object {
        "paddingLeft": 52,
        "paddingTop": 349.9285714285714,
        "position": "absolute",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "white",
          "borderRadius": 24.642857142857146,
          "width": 648,
        }
      }
    >
      <View
        style={
          Object {
            "paddingLeft": 560,
            "paddingTop": 41.07142857142857,
          }
        }
      >
        <ForwardRef
          onPress={[Function]}
        >
          <Image
            source={
              Object {
                "default": "",
              }
            }
            style={
              Object {
                "height": 31.21428571428571,
                "width": 31.21428571428571,
              }
            }
          />
        </ForwardRef>
      </View>
      <View
        style={
          Object {
            "paddingTop": 62.42857142857142,
          }
        }
      />
      <View
        style={
          Object {
            "paddingLeft": 124,
            "paddingRight": 126.00000000000001,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#1F1F60",
              "fontFamily": "BananaGrotesk-Bold",
              "fontSize": 32.85714285714286,
              "lineHeight": 44.35714285714286,
              "textAlign": "center",
            }
          }
        >
          Deactivate Account
        </Text>
      </View>
      <View
        style={
          Object {
            "paddingTop": 13.142857142857142,
          }
        }
      />
      <View
        style={
          Object {
            "paddingLeft": 64,
            "paddingRight": 62,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#1F1F60",
              "fontFamily": "BananaGrotesk-Light",
              "fontSize": 26.285714285714285,
              "lineHeight": 44.35714285714286,
              "textAlign": "center",
            }
          }
        >
          Are you sure you want to deactivate this account? You can never recover your account in the future.
        </Text>
      </View>
      <View
        style={
          Object {
            "paddingTop": 37.785714285714285,
          }
        }
      />
      <View
        style={
          Object {
            "paddingLeft": 47,
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#7B46E4",
              "borderRadius": 63.25,
              "height": 82.14285714285714,
              "width": 550,
            }
          }
        >
          <View
            style={
              Object {
                "paddingTop": 26.285714285714285,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontFamily": "BananaGrotesk-Bold",
                  "fontSize": 26.285714285714285,
                  "lineHeight": 29.571428571428573,
                  "textAlign": "center",
                }
              }
            >
              Continue
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "paddingTop": 49.28571428571429,
          }
        }
      />
    </View>
  </View>
</View>
`;
